octopoes/bits/ask_disallowed_domains/question_schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "/bit/disallowed-csp-hostnames",
+  "type": "object",
+  "default": {},
+  "Port Configuration": "Root Schema",
+  "required": [],
+  "properties": {
+    "disallowed_hostnames": {
```

Comment:
Maybe we should make sure we also allow the user to optionally deny url-shorteners instead of doing this by definition.

Reply:
Good suggestion. Will do that